src/Outline/OutlineResultPrinter.php

```diff
@@ -111,6 +123,21 @@ protected function getResultText( QueryResult $res, $outputMode ) {
 			$result .= $link->getText( $outputMode, $this->mLinker ) . "\n";
 		}
 
+
+		// Add introtemplate
+		if ( $this->params['introtemplate'] !== '' ) {
+			$introtemplate = "{{" . $this->params['introtemplate'] . "}}";
+
+			$result = $introtemplate . $result;
+		}
```

Comment:
You can avoid the comment, and a bunch of complexity, by creating a new function. 

`$result = $this->getIntroTemplate() . $result;`

```php
function getIntroTemplate(): string {
    // return empty string if param not set

    // return "{{" . $this->params['introtemplate'] . "}}"
}
```

Reply:
ok thanks for your review I will remove the comments and put the functionality to new functions.